Packaging.cmake

```diff
@@ -0,0 +1,64 @@
+if(NOT VERSION)
+	set(VERSION "unknown")
+endif()
```

Comment:
I'm not sure what's going on in awesomeConfig, but when I removed `git` (via `chmod -x`), `VERSION` was empty and hence the next line was failing with wrong number of arguments. It's possible that my own `chmod -x` leads to different behaviour than actually removing `git` from `$PATH`, though...